### PR TITLE
Add template for issues

### PR DESCRIPTION
**Pull request by docprofsky**
We should add an issue template to make it clear what is needed for reporting a bug or requesting a feature.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  1.Go to the home screen of RoboPaint, this is the screen you see when you start
+  RoboPaint and can be accessed by clicking on the home icon in the top left
+  corner of the window.
+  2.Below RoboPaint text in the top middle of the home screen you should see text
+  in parenthesizes starting with `v` copy this and put it below.
+
+RoboPaint version:
+
+#### How did you install RoboPaint?
+I did not, running from source
```
Comment (docprofsky) [inline ISSUE_TEMPLATE.md]: I considered making these check boxes, but decided against it, as GitHub interprets it as a task list.

Comment (techninja) [inline ISSUE_TEMPLATE.md]: That's what I figured.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Windows XP, Vista, 7, 8, 10
+OS X 10.?
+Linux what distribution?
+
+#### What version of RoboPaint are you running?
+  1.Go to the home screen of RoboPaint, this is the screen you see when you start
+  RoboPaint and can be accessed by clicking on the home icon in the top left
+  corner of the window.
+  2.Below RoboPaint text in the top middle of the home screen you should see text
+  in parenthesizes starting with `v` copy this and put it below.
```
Comment (docprofsky) [inline ISSUE_TEMPLATE.md]: You can not; corrected.

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+For an enhancement or new feature: Describe your needs.
+
+### Versions
+This is only needed for a bug
+
+#### What operating system are you running?
+Windows XP, Vista, 7, 8, 10
+OS X 10.?
+Linux what distribution?
```
Comment (docprofsky) [inline ISSUE_TEMPLATE.md]: Fixed

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  RoboPaint and can be accessed by clicking on the home icon in the top left
+  corner of the window.
+  2.Below RoboPaint text in the top middle of the home screen you should see text
+  in parenthesizes starting with `v` copy this and put it below.
+
+RoboPaint version:
+
+#### How did you install RoboPaint?
+I did not, running from source
+Manually
```
Comment (docprofsky) [inline ISSUE_TEMPLATE.md]: Removed

**Comment by docprofsky** [comment]
Should we use checkboxes for each question, so if we can keep track of the information we have and need?

File: ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+System installer
+Copied files from zip
+
+#### What robot are you using?
+AxiDraw
+WaterColorBoat
```
Comment (docprofsky) [inline ISSUE_TEMPLATE.md]: Aquatic robots!

File: ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+Other (specify)
+
+#### Is the robot connected to RoboPaint
```
Comment (docprofsky) [inline ISSUE_TEMPLATE.md]: Updated

**Comment by docprofsky** [comment]
Should I add manual mode to the mode listing in the README?
https://github.com/evil-mad/robopaint#projects

**Comment by techninja** [comment]
Yea, Add it in this PR. Some people DO still use it, and it has a planned future at least.

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
+# Contributing to RoboPaint
+
+Welcome, and thank you for helping us make this little project the best it can
+be!
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue, or [Gitter chat](https://gitter.im/evil-mad/robopaint) before
```
Comment (techninja) [inline .github/CONTRIBUTING.md]: Sure.

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+jumping straight in to making a change that you wish to be part of the project.
+
+Please note we have a code of conduct (at the bottom of this document), please
+follow it in all your interactions with the project.
+
+## Issues:
+RoboPaint in it's latest iteration is actually a tight grouping of smaller
```
Comment (techninja) [inline .github/CONTRIBUTING.md]: Whoops!

File: .github/CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+[the projects section of the readme](https://github.com/evil-mad/robopaint#projects).
+
+Before filling an issue or PR, please check if the bug or feature request
+already exists in the central
+[issue list](https://github.com/evil-mad/robopaint/issues).
+
+## Reporting bugs:
+
+### How to get version number:
+<div style="text-align: center;"><img src="https://cloud.githubusercontent.com/assets/320747/18774445/2742ce00-810e-11e6-85f4-d222da447309.png"></div>
```
Comment (techninja) [inline .github/CONTRIBUTING.md]: Nah. These things last forever. I mean, I guess if you want to.

File: .github/CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+To the bottom right of the RoboPaint text in the top middle of the window you
+should see the version in parenthesizes starting with `v`. This will be hidden
```
Comment (techninja) [inline .github/CONTRIBUTING.md]: I think that was from your original text!

File: .github/CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
+* Personal attacks
+* Trolling or insulting/derogatory comments
+* Public or private harassment
+* Publishing other's private information, such as physical or electronic
+addresses, without explicit permission
+* Other unethical or unprofessional conduct.
```
Comment (techninja) [inline .github/CONTRIBUTING.md]: Agreed.

File: .github/ISSUE_TEMPLATE.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
+### Description
+
+For a bug: Describe the bug and list the steps you used when the issue occurred.
+include any files necessary to reproduce this by zipping them and attaching them
```
Comment (techninja) [inline .github/ISSUE_TEMPLATE.md]: Yup.

Comment (techninja) [inline .github/ISSUE_TEMPLATE.md]: Yup.

File: .github/ISSUE_TEMPLATE.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+to this issue. For features, describe what you'd like, how it might be used and
```
Comment (techninja) [inline .github/ISSUE_TEMPLATE.md]: :ox: :car: :+1:

File: .github/ISSUE_TEMPLATE.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+why.
+
+#### Steps to reproduce | Feature Idea (Delete one)
```
Comment (techninja) [inline .github/ISSUE_TEMPLATE.md]: It's not meant to be a Boolean, just a clear separator. "||" would be the correct and I didn't do that on purpose. Regardless, I accept your suggestion as it's full english, as long as it retains the ALL CAPS for `OR`.

Comment (techninja) [inline .github/ISSUE_TEMPLATE.md]: Fine.

File: .github/ISSUE_TEMPLATE.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+  1. Click on this
+  2. Do that
+  3. ...
+
+For an enhancement or new feature: Describe your needs or wants.
+
+### Version effected (if bug)
```
Comment (techninja) [inline .github/ISSUE_TEMPLATE.md]: Sure.

File: .github/ISSUE_TEMPLATE.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+Windows: 7, 8, 10 ?
+OS X: 10.?
+Linux: Distribution (Ubuntu, Debian, RedHat, Fedora) and version
+
+#### How did you install RoboPaint?
+Running from source
+System installer
+Copied files from zip
+
+#### What robot are you using?
```
Comment (techninja) [inline .github/ISSUE_TEMPLATE.md]: Probably, though if someone has gone to the trouble of doing this, it'll be clear to them that it's different. Also it changes the version text.

File: .github/CONTRIBUTING.md (hunk 7 above)
Comment (docprofsky) [inline .github/CONTRIBUTING.md]: changed

File: .github/CONTRIBUTING.md (hunk 8 above)
Comment (docprofsky) [inline .github/CONTRIBUTING.md]: fixed

File: .github/CONTRIBUTING.md (hunk 10 above)
Comment (docprofsky) [inline .github/CONTRIBUTING.md]: Oops, fixed

File: .github/CONTRIBUTING.md (hunk 11 above)
Comment (docprofsky) [inline .github/CONTRIBUTING.md]: fixed

File: .github/ISSUE_TEMPLATE.md (hunk 12 above)
Comment (docprofsky) [inline .github/ISSUE_TEMPLATE.md]: fixed

Comment (docprofsky) [inline .github/ISSUE_TEMPLATE.md]: fixed

File: .github/ISSUE_TEMPLATE.md (hunk 13 above)
Comment (docprofsky) [inline .github/ISSUE_TEMPLATE.md]: fixed

File: .github/ISSUE_TEMPLATE.md (hunk 14 above)
Comment (docprofsky) [inline .github/ISSUE_TEMPLATE.md]: fixed

Comment (docprofsky) [inline .github/ISSUE_TEMPLATE.md]: I understand what you mean by it being a separator and not a boolean operator, but as we agree `OR` is clearer than `|` or `||`.

Changed

File: .github/ISSUE_TEMPLATE.md (hunk 15 above)
Comment (docprofsky) [inline .github/ISSUE_TEMPLATE.md]: fixed

File: .github/CONTRIBUTING.md (hunk 9 above)
Comment (docprofsky) [inline .github/CONTRIBUTING.md]: Then I will leave this.

File: .github/ISSUE_TEMPLATE.md (hunk 16 above)
Comment (docprofsky) [inline .github/ISSUE_TEMPLATE.md]: > Probably, though if someone has gone to the trouble of doing this, it'll be clear to them that it's different.

Remember #156?

> Also it changes the version text.

Technically the bot type is shown after the version string: `(v2.0.0-beta.2)-AxiDraw`
Also the bot type is hidden if it is set to `WaterColorBot`.

**Pull request merged**